scripts/build/Platform/Linux/build_config.json

```diff
@@ -58,6 +58,20 @@
       "CMAKE_TARGET": "all"
     }
   },
+  "profile_gcc_nounity": {
+    "TAGS": [
+      "nightly-incremental",
+      "nightly-clean"
```

Comment:
instead of creating these two jobs isolated, since `test_profile_gcc_nounity` will build also everything that `profile_gcc_nounity` has, create a pipe job that runs `profile_gcc_nounity` and then `test_profile_gcc_nounity`